### PR TITLE
Windows console ANSI color issue fixed

### PR DESCRIPTION
**Pull request by rupeshs**
This is quick fix for windows console color issue
windows-console-color

Fixed output
color-fixed

### CODE REVIEW

File: chat.cpp [hunk 1]
```diff
@@ -16,6 +16,7 @@
 #include <unistd.h>
 #elif defined (_WIN32)
 #include <signal.h>
+#include <Windows.h>
```
Comment (lofcz) [inline chat.cpp:19]: import conditionally

Comment (rupeshs) [reply chat.cpp:19]: It is under the preprocessor directive #elif defined (_WIN32).

File: chat.cpp [hunk 2]
```diff
@@ -886,6 +887,11 @@ int main(int argc, char ** argv) {
         sigaction(SIGINT, &sigint_action, NULL);
 #elif defined (_WIN32)
         signal(SIGINT, sigint_handler);
+		//Windows console ANSI color fix 
+		HANDLE hConsole = GetStdHandle(STD_OUTPUT_HANDLE);
```
[the remark this answers is not in the document]
Comment (rupeshs) [reply chat.cpp:891]: It is under the preprocessor directive #elif defined (_WIN32).

[the remark this answers is not in the document]
Comment (lofcz) [reply chat.cpp:891]: Oh sorry, didn't notice

**Pull request merged**